flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/ProcessingTimeServiceImpl.java

```diff
@@ -61,7 +61,7 @@ public long getCurrentProcessingTime() {
     public ScheduledFuture<?> registerTimer(long timestamp, ProcessingTimeCallback target) {
         if (isQuiesced()) {
             return new NeverCompleteFuture(
-                    ProcessingTimeServiceUtil.getProcessingTimeDelay(
+                    ProcessingTimeServiceUtil.getRecordProcessingTimeDelay(
```

Comment:
I don't think this rename is a good idea. As I mentioned in the JIRA ticket, this doesn't have to be about processing records, but about any abstract events that are happening in the system using processing time.